### PR TITLE
Feature/get list users to update

### PR DESCRIPTION
**Pull request by flopezag**
### REVIEWERS
@chemaper  @jframos 

### DESCRIPTION
- Added class to recover the list of expired users
- Added Unit Tests associated to this User Story
- Added QA acceptance test associated to this User Story

### TESTED
- Locally and externally with the FIWARE Lab

### CODE REVIEW

File: tests/acceptance_tests/component/keystone/features/keystone.feature [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+Feature: Get the list of expired users from the IdM.
+    As a fiware lab administrator
+    I want to be able to get the list of expired users from the IdM
+    so that I can delete the resources associated to them.
+
+
+    @basic
```
Comment (jframos) [inline tests/acceptance_tests/component/keystone/features/keystone.feature]: I prefer **@happy_path** tag name

File: tests/acceptance_tests/component/keystone/features/keystone_token.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    world.expiredusers.token = None
+
+@step(u'a connectivity to the Keystone service')
+def set_keystone_service_endpoint(step):
+
+    world.expiredusers.set_keystone_endpoint(KEYSTONE_URL)
+
+    try:
+        _ = requests.get(world.expiredusers.get_keystone_endpoint(), timeout=5)
+        print("True")
```
Comment (jframos) [inline tests/acceptance_tests/component/keystone/features/keystone_token.py]: I think this _print_ is not necessary.

File: expired_users.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+        headers = {'X-Auth-Token': self.token}
+        r = requests.get(url=url, headers=headers)
+
+        role_assignments = json.loads(r.text)['role_assignments']
+
+        # Extract the list of user_ids
+        for item in role_assignments:
+            self.listUsers.append(item['user']['id'])
+
+        print("Number of Trial users detected: {}".format(len(self.listUsers)))
```
Comment (chemaper) [inline expired_users.py]: It's not important, but I think that the function shouldn't print the number of users. The caller of the function may print this message if they want.

File: expired_users.py [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+    def gerlisttrialusers(self):
+        return self.listUsers
+
+    def getlistusers(self):
+        """
+        Global method that call the rest of internal one in order to recover the information of
+        the expired users.
+        :return: Lists of Expired Users id who have Trial role and expired, example:
+                    ['0f4de1ea94d342e696f3f61320c15253', '24396976a1b84eafa5347c3f9818a66a']
```
Comment (chemaper) [inline expired_users.py]: list, it is singular

File: tests/acceptance_tests/requirements.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+git+https://github.com/telefonicaid/lettuce-tools.git#egg=lettuce_tools
```
Comment (jframos) [inline tests/acceptance_tests/requirements.txt]: I can see you are using _requests_. Can you add it to `requirements` file?

**Comment by chemaper** [comment]
LGTM

**Pull request merged**